### ISSUE TITLE
The 'Yes' button is not selected on promote snapshot dialog for one blob

### ISSUE THREAD

**Issue by v-xianya** [microsoft/AzureStorageExplorer#1896]
**Storage Explorer Version:** 1.10.1
**Build:** [20190924.3](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3076815&view=logs&j=c9f4993e-ba9a-5f87-11e6-962dc1008ac4)
**Branch:** master
**Platform/OS:** Windows 10/ Linux Ubuntu 18.04/macOS High Sierra
**Architecture:** ia32/x64
**Regression From:** Previous release (1.9.0)

**Steps to reproduce:**
1. Expand one non-ADLS Gen2 account -> Blob Containers. 
2. Create a new blob container -> Upload a blob.
3. Create snapshot for the blob -> Manage snapshots.
4. Promote snapshot -> Check the prompt dialog.

**Expect Experience:**
The 'Yes' button is selected.
![image](https://user-images.githubusercontent.com/41351993/65506399-0c3e9c00-defe-11e9-9f7e-9e971e15cc89.png)

**Actual Experience:**
The 'Yes' button is not selected.
![image](https://user-images.githubusercontent.com/41351993/65506409-119be680-defe-11e9-9a6c-32d1ae9d1eee.png)

**More Info:**
This issue also reproduces when trying to open a file which is potentially unsafe.
![image](https://user-images.githubusercontent.com/41351993/65506957-612ee200-deff-11e9-93d1-4b0eaf80cbae.png)

**Comment by craxal** [comment]
@jinglouMSFT Do we want to select the Yes button by default? I intentionally omitted this since an overwrite can result in data loss, and I didn't want users to accidentally move forward with an operation like that.

**Comment by MRayermannMSFT** [comment]
1. Regardless of the action being executed, there should be a default button
2. Typically, for "data loss" type warnings, Yes is the default. For "security" type warnings, No is the default. At least in my experience.

**Comment by craxal** [comment]
Let us verify. We may need to update some module references.

**Comment by v-limu** [comment]
This issue reproduces on build [20191213.1](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3318376). So we reopen it.
![image](https://user-images.githubusercontent.com/54055206/70793291-f0001c00-1dd5-11ea-8140-7721c3791345.png)

**Comment by craxal** [comment]
@v-limu This bug was fixed for 1.12.0. Build 20191213.1 is based on a hotfix for 1.11.1, which doesn't have this fix, so I'm closing this issue again. If it repros in master builds, please reopen then.
